index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+      <nav>
+          <p class="logo">My Logo</p>
+          <ul>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">about</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+          <div class="menu">
+            <img src="./images/Icon - Menu.png" alt="">
+            <a href="toogle-nav">Menu</a>
+          </div>
+      </nav>
+      <header>
+        <div class="topimg">
+            <div class="imgorder imgorder1">
+              <img class="backimg backimg1" src="./images/Ellipse 11.png" alt="">
+              <img class="backimg backimg2" src="./images/Rectangle 4.png" alt="">
+            </div>
+            <img src="" alt="">
+            <div class="imgorder imgorder2">
+              <img class="backimg backimg3" src="./images/Rectangle 7.png" alt="">
+              <img class="backimg backimg4" src="./images/Ellipse 5.png" alt="">
+            </div>
+            <img class="icon" src="./images/Ellipse 7.png" alt="">
+            <div class="imgorder imgorder3">
+              <img class="backimg backimg5" src="./images/Rectangle 4.png" alt="">
+              <img class="backimg backimg6" src="./images/Ellipse 3.png" alt="">
+            </div>
```

Comment:
- [ ] Sorry this comment wasn't addressed by the previous reviewer but I highly suggest using CSS styling background image instead of the img tag, since these images are only for decorative purposes.

for more information please follow [When to Use HTML <img> Tag and CSS background-image Property](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html)